bedrock/firefox/tests/test_base.py

```diff
@@ -324,6 +324,14 @@ def test_fx_nightly_68_0_a1_whatsnew(self, render_mock):
         template = render_mock.call_args[0][1]
         assert template == ["firefox/nightly/whatsnew.html"]
 
+    @override_settings(DEV=True)
+    def test_fx_nightly_100_0_a1_whatsnew(self, render_mock):
+        """Should show nightly whatsnew template"""
+        req = self.rf.get("/en-US/firefox/whatsnew/")
+        self.view(req, version="100.0a1")
```

Comment:
Minor, non-blocking suggestion: might be worth `parametrize`-ing this and the Dev Version test so we can be sure we're OK for two-digit and three-digit versions

Reply:
This is probably a limit of my Python knowledge, but I'm not actually sure how to use Pytest's parameterize markers inside a class based test set up. I can't really find any examples that do this, so I'm not sure how to accomplish this without at least a substantial test refactor. Maybe this is something you would like to take on in another issue, if you still think it's worth doing?

The Dev Edition test directly below this one already tests using a two digit version number, so from a coverage perspective I think we're still OK here. I'm going to merge this, but I'm curious how you would solve this for my own learning perspective 🙂